### DISCUSSION TITLE
code-server: not found

### DISCUSSION THREAD

**Discussion by zichunxx** [coder/code-server#6832] [Q&A]
Hi!

I tried to install code-server via Dockerfile with following snippet:
```
RUN curl -fsSL https://code-server.dev/install.sh | sh
RUN code-server --install-extension ms-python.python ms-toolsai.jupyter
```

However, I got this error report:

```
 > [16/31] RUN code-server --install-extension ms-python.python ms-toolsai.jupyter:
0.312 /bin/sh: 1: code-server: not found
------
Dockerfile:96
--------------------
  94 |     # install VS Code (code-server)
  95 |     RUN curl -fsSL https://code-server.dev/install.sh | sh
  96 | >>> RUN code-server --install-extension ms-python.python ms-toolsai.jupyter
  97 |     
  98 |     # install VS Code extensions
--------------------
ERROR: failed to solve: process "/bin/sh -c code-server --install-extension ms-python.python ms-toolsai.jupyter" did not complete successfully: exit code: 127
"docker run" requires at least 1 argument.
See 'docker run --help'.

Usage:  docker run [OPTIONS] IMAGE [COMMAND] [ARG...]

Create and run a new container from an image
```

Why code-server command is not found? Thanks!

**Comment by code-asher** [comment]
Could you post the output of the `curl` command too?

**Reply by zichunxx** [reply]
Hi! 

~~The `curl` command works fine without any output when building Dockerfile. The above error report is caused by `RUN code-server --install-extension ms-python.python ms-toolsai.jupyter`.~~

Sorry for the carelessness. I found the output of the curl command:

`1.121 curl: (7) Failed to connect to raw.githubusercontent.com port 443: Connection refused`. However, I have used the proxy.

**Reply by code-asher** [reply]
Are you using the `http_proxy` and `https_proxy` environment variables?  cURL should pick those up, yeah.  You could also try `curl --proxy` to explicitly use a proxy.
